roles/windows/microsoft_tools/tasks/main.yml

```diff
@@ -1,3 +1,9 @@
+- name: Register a variable
```

Comment:
do you need these in every task?

Reply:
Thats a good question, I dont think I tested it without, let me make some attempts real quick with registering it once